Task_DEV_3/Task_DEV_3/Program.cs

```diff
@@ -4,30 +4,41 @@ namespace Task_DEV_3
 {
     class Program
     {
+        static int[] CreateNumbersArray(int arrayLength)
+        {
+            int[] inputNumbers = new int[arrayLength];
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine($"Enter an array element № {i}");
+                inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
+            }
+            return inputNumbers;
+        }
+
+        static bool CheckNumbersForPrime(int number)
+        {
+            bool isPrimeNumber = true;
+            for (int j = 2; j < number; j++)
+            {
+                if (number % j == 0)
+                {
+                    isPrimeNumber = false;
+                    break;
+                }
+            }
+            return isPrimeNumber;
+        }
+
         static void Main(string[] args)
         {
             Console.WriteLine("Enter the length of the array :");
             int arrayLength = Convert.ToInt32(Console.ReadLine());
             if (arrayLength <= 5)
             {
-                int[] inputNumbers = new int[arrayLength];
-                for (int i = 0; i < arrayLength; i++)
-                {
-                    Console.WriteLine($"Enter an array element № {i}");
-                    inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
-                }
-                bool isPrimeNumber;
+                int[] inputNumbers = CreateNumbersArray(arrayLength);
                 for (int i = 0; i < arrayLength; i++)
                 {
-                    isPrimeNumber = true;
-                    for (int j = 2; j < inputNumbers[i]; j++)
-                    {
-                        if (inputNumbers[i] % j == 0)
-                        {
-                            isPrimeNumber = false;
-                            break;
-                        }
-                    }
+                    bool isPrimeNumber = CheckNumbersForPrime(inputNumbers[i]);
```

Comment:
You don't even need to create a separate variable `isPrimeNumber`. It is possible to do smth like:
`if(CheckNumbersForPrime(inputNumbers[i]))`

Reply:
Done.